jenkins-scripts/docker/lib/gazebo-base-default.bash

```diff
@@ -134,7 +134,7 @@ cat >> build.sh << DELIM_BUILD_DEPS
       dependency_repo="osrf/${dep}"
     else
       # need to replace _ by -
-      dependency_repo="ignitionrobotics/${dep/_/-}"
+      dependency_repo="gazebosim/${dep/_/-}"
```

Comment:
Here too

Reply:
![image](https://user-images.githubusercontent.com/20265670/175456032-1a378a45-665c-40d4-9175-5dfaf453d92d.png)

It's just used for github cloning! Should be safe